packages/venia-concept/src/components/Checkout/addressForm.js

```diff
@@ -132,6 +154,9 @@ class AddressForm extends Component {
                             className={classes.textInput}
                         />
                     </div>
+                    <div className={classes.validation}>
+                        {this.validationBlock(formState.errors)}
+                    </div>
```

Comment:
If there are no validation errors this will output an empty div, which may or may not be what you want.

An alternative is to have a getter that returns this JSX or `null` and then just render the getter.

Reply:
Having empty div allows us to retain space for potential errors and when they will occur the layout will not jump

Reply:
Agreed, if the layout jumps that's no good. This is not a huge deal 👍